Sources/PrinceOfVersions/Objective-C Helpers/Objective-C Extensions/ObjectiveCBaseExtensions.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  PoVObjC.swift
+//  PrinceOfVersions
+//
+//  Created by Jasmin Abou Aldan on 13/09/2019.
+//  Copyright © 2019 Infinum Ltd. All rights reserved.
+//
+// Used for exposing Swift methods to the ObjectiveC
+// As we don't want to show duplicated methods (one for Swift and one for ObjC)
+// simple @available will be used for hiding wrapper methods in Swift.
+
+import Foundation
+
+@objcMembers
+public class UpdateResponse: NSObject {
```

Comment:
Should we hide this for swift?

Reply:
Even though this class is written to be used with Objective-C PoV methods, exposed Swift methods to Objc are still written in Swift so we cannot hide this.